src/theme/Navbar/index.tsx

```diff
@@ -1,14 +1,107 @@
-import type NavbarType from '@theme/Navbar';
-import type { WrapperProps } from '@docusaurus/types';
-import React from 'react';
-import Navbar from '@theme-original/Navbar';
+import React from "react";
+import NavbarLayout from "@theme/Navbar/Layout";
+import NavbarContent from "@theme/Navbar/Content";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import Logo from "../../../images/polykey-logo-light.svg";
 
-type Props = WrapperProps<typeof NavbarType>;
-
-export default function NavbarWrapper(props: Props): JSX.Element {
+export default function Navbar() {
+  const [dynamicClass, setDynamicClass] = React.useState(false);
+  const [email, setEmail] = React.useState("");
   return (
-    <>
-      <Navbar {...props} />
-    </>
+    <NavbarLayout>
+      <div className="container">
+        <div className="header_main">
+          <div className="header_logo">
+            <img src="https://uploads-ssl.webflow.com/6098c4e3255a0a67635c1de5/6098c4e3255a0ac4e35c1e05_poly-key-logo.svg" />
+          </div>
+          <div className="header_menu">
+            <ul>
+              <li>
+                <a href="https://polykey.io/blog">Blog</a>
+              </li>
+              <li>
+                <a href="https://polykey.io/docs">Docs</a>
+              </li>
+              <li>
+                <a href="https://polykey.io/download">Download</a>
+              </li>
+              <li>
+                <a href="https://github.com/MatrixAI/PolyKey/discussions">
+                  <img
+                    src="https://uploads-ssl.webflow.com/6098c4e3255a0a67635c1de5/6098c4e3255a0a39ef5c1e04_git-hub-white.svg"
+                    alt=""
+                  />
+                </a>
+              </li>
+              <li className="email_box">
+                <input
+                  type="email"
+                  placeholder="Your Email"
+                  onChange={(e) => {
+                    setEmail(e.target.value);
+                  }}
+                />
+                <button
+                  onClick={() => {
+                    console.log(email);
+                  }}
+                >
+                  Get Started
+                </button>
+              </li>
```

Comment:
This is supposed to be a `form` element.

![image](https://user-images.githubusercontent.com/640797/201471183-9c9f1a85-1b03-4d16-a5dd-450e2ae1cd93.png)

Plus you need to have the form action so the form actually works. Right now this form element doesn't do anything!

Reply:
See `polykey.io` view the source, and copy the HTML form.

Reply:
https://imgur.com/DDyyMPq